test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git.
```

Comment:
- [ ] You should ignore the `test.md ` in `.gitignore`, please create a file called `.gitignore` and then ignore the test.md file inside it.  😉